### DISCUSSION TITLE
`project: type: book` project organization, but `scrartl`-styled article output?

### DISCUSSION THREAD

**Discussion by r-cheologist** [quarto-dev/quarto-cli#3154] [Q&A] [answered]
Maybe this is turning into the central discussion hub regarding `quarto`, so I re-post [this](https://stackoverflow.com/q/74275817/2103880) ... please forgive me, but I prefer the stackoverflow facilities to those here...

I'm looking to be able to keep the `project: type: book`-inherent project organization with multiple `*.qmd` files being merged into a single (PDF)output and in particular the option to manage individual `*.qmd` inclusion via `book: chapters: *` definitions in profile-specific `_quarto-<PROFILENAME>.yml` files, **BUT** use KOMA-Script's `scrartcl` rather than the default `screprt` as the document class.

This is hindered by the fact that `quarto` uses LaTeX code not supported by `scrartcl` (like `chapter`) - of which I cannot even discern the origin of, as none of the template snippets [here](https://github.com/quarto-dev/quarto-cli/tree/main/src/resources/formats/pdf/pandoc) contain them.

Any idea on how to achieve this? Is it feasible?

**Comment by dragonstyle** [comment, marked as answer]
I can't really think of a way to accomplish this by simply switching classes. There are a variety of options for customizing the behavior of the KOMA templates (e.g. `oneside`, `title page`) that might get you closer to what you're looking for. You could also trying redefining the `chapter` command - I'm not sure how well that would work but just throwing it out there.

**Comment by r-cheologist** [comment]
Very good point. I have set out to adapt, rather than trying to shift
away from `scrartcl`. Let's see how far I get.

Joh
